app/models/setting.rb

```diff
@@ -201,6 +201,7 @@ def parse_string_value(val)
   def self.create(opts)
     # self.name can be set by default scope, e.g. from first_or_create use
     opts = { :name => new.name } if opts.nil?
+    opts.symbolize_keys!
```

Comment:
nitpicking, why not do it like so:

```
if opts.nil? 
  opts =  { :name => new.name } 
else
   opts.symbolize_keys!
end
```

When you create the new hash, you already know that it has symbols for name, so it just shout in my head due to OCD :D